lib/davinci_pas_test_kit/generator/templates/suite.rb.erb

```diff
@@ -47,12 +47,15 @@ module DaVinciPASTestKit
 
       input :smart_credentials,
             title: 'OAuth Credentials',
-            type: :oauth_credentials,
+            type: :auth_info,
+            options: {
+              mode: 'access'
+            },
             optional: true
```

Comment:
Any way to make this just say "Auth Type" instead of "smart_credentials Auth Type"?

Reply:
it will look better. That's a question for @AlyssaWang . I believed the way it was implemented was bay concatenation the the input name and 'Auth Type'. This change will happen core.

Reply:
Gotcha, wasn't sure if it needed a core change, so thanks for confirming.